types/src/encode.rs

```diff
@@ -0,0 +1,27 @@
+pub struct Encoder<'a>(pub &'a mut dyn alloy_rlp::BufMut);
```

Comment:
What's `Encoder` for?

Reply:
Nothing, I were using it but found a way without him (use alloy_rlp::RlpEncodableWrapper instead RlpEncodable)